website/static/js/logActionsList.json

```diff
@@ -61,6 +61,7 @@
     "retraction_initiated":     "${user} initiated withdrawal of registration of ${node}",
     "retraction_initiated_no_user": "A withdrawal of registration of ${node} is proposed",
     "retraction_approved":  "${user} approved withdrawal of registration of ${node}",
+    "retraction_approved_no_user":  "A withdrawal of registration of ${node} approved",
```

Comment:
This isn't actually being used, so I would remove? It seems like you are copying similar patterns for other logs (I don't think those logs are correct).

The `retraction_approved` log above is used for both cases, where there is a user or not.  If there's no user, `logTextParser.js` takes care of making user be `A user`.
<img width="613" alt="Screen Shot 2019-07-22 at 10 09 07 AM" src="https://user-images.githubusercontent.com/9755598/61643570-c698f500-ac68-11e9-8890-4d522f6ee131.png">

Reply:
Nope, I was wrong about this, @corbinSanders pointed me to this line, which uses this log action. Yikes.  Honestly, I don't know what value this extra log adds?
```js 
var newAction = logObject.attributes.action + '_no_user'; return logActions[newAction] ? logActions[newAction]: text;
```